tests/fast/RocksCycleTest.toml

```diff
@@ -0,0 +1,46 @@
+[configuration]
+storageEngineType = 4 # Always pick RocksDB
+
+[[knobs]]
+rocksdb_metrics_in_simulation = true # This can cause non-determinism, but we don't do unseed check for this test
+
+[[test]]
+testTitle = 'Clogged'
+
+[[test.workload]]
+testName = 'RocksCycle'
```

Comment:
How different is this workload from Cycle? Is it just `noUnseed = true;`?

Reply:
Mostly the same, `RocksCycle.actor.cpp` also mentions that in the comment. The main requirements are:
- set noUnseed to true
- set some knobs to true
- have a separate test and not touch the original one

I tried reusing Cycle.actor.cpp` logic in this test but there were some challenges: no header file, did not want tenant logic, etc. Any suggestion on how to reuse code better?

Reply:
If the only differences are `noUnseed` and knobs, they can be specified in the TOML file. We only need to add `noUnseed` as a new test parameter for non-deterministic tests.

Another way is to create a header file for Cycle workload and inherit from that workload. However, I think TOML file specification is a better approach and can be reused for other tests.

Reply:
I did consider reusing workload code with a new TOML file. The challenge was how to force rocksdb storage engine in this approach (sorry, forgot to mention this difference earlier).

Reply:
Never mind, I was thinking something else, your approach should work. Let me try.

Reply:
Done. Looks much better, thanks for the feedback.